tutorials/asr/Online_Noise_Augmentation.ipynb

```diff
@@ -752,8 +752,9 @@
             "source": [
                 "# This is where the rir data will be downloaded.\n",
                 "# Change this if you don't want the data to be extracted in the current directory.\n",
+                "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/scripts/dataset_processing/get_openslr_rir_data.py\n",
                 "rir_data_path = '.'\n",
-                "!python ../../scripts/dataset_processing/get_openslr_rir_data.py --data_root {rir_data_path}\n",
+                "!python get_openslr_rir_data.py --data_root {rir_data_path}\n",
```

Comment:
why not reuse the script under nemo, such as `!python ../../../scripts/dataset_processing/get_openslr_rir_data.py --data_root {rir_data_path}`?

Reply:
hmm...i did not find `scripts` folder there.

Reply:
my bad, the original `../../` should lead to NeMo's root folder, which means `../../scripts/dataset_processing/get_openslr_rir_data.py` is a valid file

Reply:
thanks! but again, it still `failed` when running through google colab. It's not a big problem though. Let's wait next QA's report then.

```
python3: can't open file '/content/../../scripts/dataset_processing/get_openslr_rir_data.py': [Errno 2] No such file or directory
head: cannot open './processed/rir.json' for reading: No such file or directory
```

Reply:
maybe some env vars are wrong on collab, but the fix should work